worker/pkg/zip/zip_test.go

```diff
@@ -0,0 +1,173 @@
+package zip
+
+import (
+	"archive/zip"
+	"bytes"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testFilePrefix = "test"
+	fileContent    = "Hello"
+)
+
+// createDummyFiles creates dummy files with given filenames
+func createDummyFiles(t *testing.T, fileNames ...string) {
```

Comment:
Unzipper only needs io.ReadAt, so bytes.Buffer is sufficient without creating a file. Creating a file adds an extra external factor that prevents a successful test, so if you don't need it, refrain from doing so.

Also, although not applicable in this case, afero.FS can be substituted in many cases if you wish to test emulating a file system.

Reply:
I've fixed [refine test case](https://github.com/reearth/reearth-cms/pull/205/commits/03ceb2728dd414243d62a87796fbfd5024f7c198)